modules/serverless/pages/serving/serving-transport-encryption.adoc

```diff
@@ -0,0 +1,446 @@
+= {serverlessproductname} Serving Transport Encryption
+:compat-mode!:
+// Metadata:
+:description: {serverlessproductname} Serving Transport Encryption
+
+{serverlessproductname} Serving transport encryption feature allows transporting data over secured and encrypted HTTPS connections using TLS.
+
+[IMPORTANT]
+====
+{serverlessproductname} transport encryption for Serving is a Developer Preview feature only.
+Developer Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Developer Preview features, see https://access.redhat.com/support/offerings/devpreview/.
+====
+
+
+
+== Overview
+
+There are three parts to {serverlessproductname} Serving Transport Encryption:
+
+image::serving-encryption/encryption-overview.drawio.svg[]
+
+1. Transport Encryption on the ingress layer external to the cluster (e.g. cluster external domain, like `myapp-<namespace>.example.com`).
+2. Transport Encryption on the ingress layer internal to the cluster (e.g. cluster local domains, like `myapp.<namespace>.svc.cluster.local`).
+3. Transport Encryption between Knative internal components (ingress-gateway, activator, queue-proxy).
+
+[IMPORTANT]
+====
+Currently, all control-plane traffic (including Kubernetes PreStopHooks, metadata and metrics) is not (yet) encrypted.
+As this contains no user data, it is considered to be implemented in a future release.
+====
+
+=== The parts in detail
```

Comment:
The wording of the title sounds a bit weird. What's about just committing it ? This would also clean up the rendered TOC above. I don't think this level-3 heading is required here.

Reply:
You then could move the level-4 headings to level-3 heading. Extra bonus: They would appear in the top-page TOC, too.